crates/provider-sdk/src/provider_main.rs

```diff
@@ -99,6 +99,7 @@ where
             }
         },
     )
+    .request_timeout(None)
```

Comment:
Shouldn't this be `Some(host_data.default_rpc_timeout_ms)`?

Reply:
Setting it to None is related to https://github.com/wasmCloud/wasmCloud/pull/1439. If we're using send_request, then it's more appropriate to set it to default_rpc_timeout_ms, and I'll change it there.